<launch>

    <arg name="ns" default="joy"/>

    <!-- navigation -->
    <node pkg="bender_joy" type="joy_navigation.py" name="joy_navigation" output="screen" ns="$(arg ns)">
        <rosparam file="$(find bender_joy)/config/joystick_navigation.yaml" command="load"/>
        <remap from="joy" to="navigation"/>
    </node>

</launch>
